src/background.js

```diff
@@ -6,7 +6,7 @@ const TRACKING_ID = 'UA-35433268-79';
 const KINTO_SERVER = 'https://kinto.dev.mozaws.net/v1';
 // XXX: Read this from Kinto fxa-params
 const FXA_CLIENT_ID = 'c6d74070a481bc10';
-const FXA_OAUTH_SERVER = 'https://oauth-scoped-keys.dev.lcip.org/v1';
+const FXA_OAUTH_SERVER = 'https://oauth-scoped-keys-oct10.dev.lcip.org/v1';
```

Comment:
@Natim @glasserc please update the kinto dev server to use this now. If this server is not working for you while I'm on PTO please rollback this commit :)

Reply:
This has the fixes to the `kid` and other stuff